src/test/scala/uk/gov/hmrc/test/ui/pages/section6/TransportCountryPage.scala

```diff
@@ -37,4 +52,8 @@ object TransportCountryPage extends BasePage {
     fillDropdown("transport-country", values(country))
     store(TransportCountry -> Detail(values(country)))
   }
+
+  def gotoDepartureTransportPage: Boolean =
+    (isStandard || isSupplementary) && ((!isFixedTransport || !isPostalOrMail || isGuernseyOrJerseyDestination) && isBorderLocation)
```

Comment:
Uhm... the logic in `!isFixedTransport || !isPostalOrMail` is suspicious

Reply:
Yes we need this logic